### ISSUE TITLE
Beams > 1 Causes Batch to Have Similar Captions

### ISSUE THREAD

**Issue by brandontrabucco** [brandontrabucco/indigo#6]
This is likely caused by an indexing error when propagating older beams into the future inside pointer.py or logits.py, and there is likely a simple fix for this error.

**Comment by brandontrabucco** [comment]
the fix was simple; see beam_search.py
